engine/common/requester/engine.go

```diff
@@ -415,7 +415,7 @@ func (e *Engine) dispatchRequest() (bool, error) {
 
 	err = e.con.Unicast(req, providerID)
 	if err != nil {
-		return true, fmt.Errorf("could not send request: %w", err)
+		return true, fmt.Errorf("could not send request for entities %v: %w", logging.IDs(entityIDs), err)
```

Comment:
This change would be useful for answering questions like why EN didn't receive the collection, did it fail to send the request for the collection?

Now since we log the entity ID, if we look for log for the collection ID on execution node, we could know whether it failed to request the collection